voxpupuli-test.gemspec

```
@@ -20,13 +20,13 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'facterdb', '~> 3.1'
   s.add_runtime_dependency 'metadata-json-lint', '~> 4.0'
   s.add_runtime_dependency 'parallel_tests', '~> 4.2'
-  s.add_runtime_dependency 'puppetlabs_spec_helper', '~> 8.0'
+  s.add_runtime_dependency 'puppet_fixtures', '~> 0.1'
+  s.add_runtime_dependency 'puppet-syntax', '~> 4.1', '>= 4.1.1'
```

I'm never a fan of such tight restrictions. Is there a reason we can't work with older versions?
```suggestion
  s.add_runtime_dependency 'puppet-syntax', '~> 4.0'
```